contrib/cirrus/lib.sh.t

```diff
@@ -4,6 +4,8 @@
 #
 source $(dirname $0)/lib.sh
 
+set -o pipefail
```

Comment:
Seems kind of fragile. I did spend some time thinking of the cleanest way to handle the asterisks line; my suggestion has the advantage of simplicity and no action-at-a-distance.

Reply:
Yeah I spent some time on it myself.  Couldn't find anything better for filtering.  I ended up using your asterisks solution, along with the pipefail.  Thx again for the help.